src/components/dialogs/customMuiDialog/CustomMuiDialog.tsx

```diff
@@ -40,6 +41,18 @@ const styles = {
             margin: 'auto',
         },
     },
+    dialogPaperFullHeight: {
+        '.MuiDialog-paper': {
+            width: 'auto',
+            minWidth: '1100px',
+            margin: 'auto',
+            height: '95vh',
```

Comment:
- Height depends on navigator window height (i thought in specs that it should be fix).
- when i reduce navigator window height i can't access content any more there should be a scrollbar no ?
![Capture d’écran de 2024-10-09 11-29-42](https://github.com/user-attachments/assets/e8d9e6a8-066f-438d-b234-e970fd50cd41)

- when i reduce navigator window width, i can't see anything
![Capture d’écran de 2024-10-09 11-33-07](https://github.com/user-attachments/assets/f2389ea5-1ebb-4a92-a962-749a14e8b59c)

Reply:
To be careful, this component is used in different creation dialogues, e.g create study, etc.

Reply:
**To Thang** : don't worry I added a param for my own version of this dialog box : other uses of this dialog box won't be affected.

**To Bassel** : I spoke with Pauline and Stéphane : The viewport of the user should be at least 1024x880 : we don't care about smaller resolution. minWidth 1100 px is a bit too big though. I will reduce it a little bit (only for my own version).

And what specs are you talking about ? Can you share ? I don't know any reason not to use window height percentage. This is used elsewhere (for the transformers modification I think).

Reply:
minWidth changed.

Reply:
Ok I did.
But not sure that this is the right move : it may be better to have small margins on both sides at all times.